src/css/support.js

```diff
@@ -102,14 +102,23 @@ define( [
 		// set in CSS while `offset*` properties report correct values.
 		// Behavior in IE 9 is more subtle than in newer versions & it passes
 		// some versions of this test; make sure not to make it pass there!
+		//
+		// Support: Firefox 70+
+		// Only Firefox includes border widths
+		// in computed dimensions. (gh-4529)
```

Comment:
That means we need to add a version of that test on `master` as well, don't we? Perhaps smaller as we don't care for Edge there.

Reply:
Good point. I'll make a PR for master as well.